### PR TITLE
Fix IOS-1121: Use Pointzi-SDK-iOS branch-commit as TestApp's version

### PR DESCRIPTION
**Pull request by ChristineXYS**
Update git_branch+git_commit_has+build_time in Test App’s build number field.

### CODE REVIEW

File: Example_StaticLibrary/PointziDemo/Supporting Files/Info.plist [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>IOS1121_GitVersion-e1d7a67b-Wed Sep 20 14:34:28 CST 2017</string>
```
Comment (jagguli) [inline Example_StaticLibrary/PointziDemo/Supporting Files/Info.plist]: this string is fixed ?

File: BuildInfo.plist [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>GitBranch</key>
+	<string>IOS1121_GitVersion</string>
```
Comment (jagguli) [inline BuildInfo.plist]: this string is fixed?

File: Example_StaticLibrary/PointziDemo/Supporting Files/Info.plist (hunk 1 above)
Comment (ChristineXYS) [reply Example_StaticLibrary/PointziDemo/Supporting Files/Info.plist]: No, it's updated with each build. 
When CI runs, this repository will dirty with new pod binary, version etc.

File: BuildInfo.plist (hunk 2 above)
Comment (ChristineXYS) [reply BuildInfo.plist]: Same as above.

**Pull request merged**